### ISSUE TITLE
libgig fails to compile in msys_helper.sh

### ISSUE THREAD

**Issue by UnityParadox** [LMMS/lmms#3518]
Under branch stable-1.2
System: Intel i5-6400 with Windows 10, Creator Update

![image](https://cloud.githubusercontent.com/assets/27585517/25557491/52f7e138-2ce0-11e7-97b5-f6e51e9cf59f.png)
![image](https://cloud.githubusercontent.com/assets/27585517/25557493/5ded7d28-2ce0-11e7-87f7-ae5d64a0c09a.png)

*@tresf edited, added searchable keywords*
```
Sorry, you need either libsndfile or libaudiofile to compile libgig.
This is needed for the gigextract utility to write audio files.
make : *** No targets specified and no makefile found.  Stop.

- Installing libgig... 
make:  Nothing to be done for 'install'.
mv: cannot stat '/mingw64/lib/bin/libakai-0.dll': No such file or directory
mv: cannot stat '/mingw64/lib/bin/libgig-7.dll': No such file or directory
```

**Comment by Umcaruje** [comment]
Please add the information about your system, and the branch you're compiling

**Comment by UnityParadox** [comment]
Information added.

**Comment by tresf** [comment]
I think the relevant portion is:
```
Sorry, you need either libsndfile or libaudiofile to compile libgig.(edited)
```

This dependency should have been met when `mingw64-x-libsndfile` was installed via the [`fetch_ppa.sh`](https://github.com/LMMS/lmms/blob/stable-1.2/cmake/msys/fetch_ppa.sh) script.   `fetch_ppa.sh` is [called automatically by `msys_helper.sh`](https://github.com/LMMS/lmms/blob/stable-1.2/cmake/msys/msys_helper.sh#L39) so as long as `$HOME/ppa` doesn't exist.

So if you don't mind re-installing the dependencies, I recommend wiping out `$HOME/ppa` and trying that script again.

**Comment by tresf** [comment]
Closed via 2f51062.

**Comment by tresf** [comment]
Typo.  Closed via f9e01b5. Thanks @edition-ben for the fix.  Cherry-picked from #3492.

`cd ~; rm -rf lmms ppa; ./msys_helper.sh` and you should be all set.
